software/open-quartermaster.yml

```diff
@@ -0,0 +1,19 @@
+name: "Open QuarterMaster"
+website_url: "https://openquartermaster.com/"
+source_code_url: "https://github.com/Epic-Breakfast-Productions/OpenQuarterMaster"
+description: "A powerful inventory management system, designed to be flexible and scalable."
+licenses:
+  - GPL-3.0
+platforms:
+  - Java
+  - Python
+  - PHP
+  - Deb
+  - Docker
```

Comment:
```suggestion
```
Same with docker 👍🏻 unless docker is a supported installation method, don't see where it might be.

Reply:
Everything runs with containers under the hood. We intend to add a deployment option for docker compose, and are working on helm charts for k8s. Up to you though.

Reply:
Yeah, keep it if you plan on it then 👍🏻 :)